### PR TITLE
MobileNet V2 번역

### PR DESCRIPTION
**Pull request by mindongjun**
## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/PyTorchKorea/hub-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #84

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [x] 번역되지 않은 모델 소개를 번역하는 기여
- [ ] 공식 허브의 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._
https://github.com/PyTorchKorea/hub-kr/blob/master/pytorch_vision_mobilenet_v2.md
mobilenet v2 모델에 대한 번역이 추가 됩니다.

### CODE REVIEW

File: pytorch_vision_mobilenet_v2.md [hunk 1]
```diff
@@ -50,46 +49,46 @@ preprocess = transforms.Compose([
...
 top5_prob, top5_catid = torch.topk(probabilities, 5)
 for i in range(top5_prob.size(0)):
     print(categories[top5_catid[i]], top5_prob[i].item())
 ```
 
-### Model Description
+### 모델 설명
 
-The MobileNet v2 architecture is based on an inverted residual structure where the input and output of the residual block are thin bottleneck layers opposite to traditional residual models which use expanded representations in the input. MobileNet v2 uses lightweight depthwise convolutions to filter features in the intermediate expansion layer. Additionally, non-linearities in the narrow layers were removed in order to maintain representational power.
+MobileNet v2 구조는 residual 블록의 입력 및 출력이 입력단에서 확장 표현을 사용하는 기존의 residual 모델과 반대되는 얇은 병목 계층인 반전된 residual 구조를 기반으로 합니다. MobileNet v2는 경량화된 depthwise 합성곱를 사용하여 중간 확장 계층의 특징 들을 필터링합니다. 또한, 표현력 유지를 위해 좁은 계층의 비선형성은 제거되었습니다.
```
Comment (euroversedev) [inline pytorch_vision_mobilenet_v2.md:85]: residual block에 대한 용어를 정의하면 좋을 것 같아요!
(여러 블로그 포스트에서 잔차블록, 잔여블록 등으로 설명)
추가로, summary와 위 문장에서 block에 대한 번역을 통일하면 좋겠습니다.

**Review by euroversedev — COMMENTED** [review]
안녕하세요😁 어려운 내용이지만, 번역이 매끄럽고 좋은 것 같습니다.
GoodGIF

File: pytorch_vision_mobilenet_v2.md [hunk 2]
```diff
@@ -23,23 +23,22 @@ model = torch.hub.load('pytorch/vision:v0.10.0', 'mobilenet_v2', pretrained=True
 model.eval()
 ```
 
-All pre-trained models expect input images normalized in the same way,
-i.e. mini-batches of 3-channel RGB images of shape `(3 x H x W)`, where `H` and `W` are expected to be at least `224`.
-The images have to be loaded in to a range of `[0, 1]` and then normalized using `mean = [0.485, 0.456, 0.406]`
-and `std = [0.229, 0.224, 0.225]`.
+모든 사전 훈련된 모델들은 입력 이미지가 동일한 방식으로 정규화되었다고 상정합니다.
+즉, 미니 배치(mini-batch)의 3-채널 RGB 이미지들은 `(3 x H x W)`의 형태를 가지며, 해당 `H`와 `W`는 최소 `224` 이상이어야 합니다.
```
[the remark this answers is not in the document]
Comment (mindongjun) [reply pytorch_vision_mobilenet_v2.md]: 안녕하세요. @euroversedev 님 번역 리뷰 감사합니다.
해당 용어 사용 규칙 확인했으며 해당 내용으로 삭제하고 수정했습니다.

File: pytorch_vision_mobilenet_v2.md (hunk 1 above)
Comment (mindongjun) [reply pytorch_vision_mobilenet_v2.md]: residual 관련하여 많이 사용되는 '잔차'로 통일했으며
추가로 리뷰주신 summary부분에도 번역 통일했습니다.
리뷰 감사합니다.

File: pytorch_vision_mobilenet_v2.md [hunk 3]
```diff
@@ -50,46 +49,46 @@ preprocess = transforms.Compose([
...
 top5_prob, top5_catid = torch.topk(probabilities, 5)
 for i in range(top5_prob.size(0)):
     print(categories[top5_catid[i]], top5_prob[i].item())
 ```
 
-### Model Description
+### 모델 설명
 
-The MobileNet v2 architecture is based on an inverted residual structure where the input and output of the residual block are thin bottleneck layers opposite to traditional residual models which use expanded representations in the input. MobileNet v2 uses lightweight depthwise convolutions to filter features in the intermediate expansion layer. Additionally, non-linearities in the narrow layers were removed in order to maintain representational power.
+MobileNet v2 구조는 잔차 블록의 입력 및 출력이 입력단에서 확장 표현을 사용하는 기존의 잔차 모델과 반대되는 얇은 병목 계층인 반전된 잔차 구조를 기반으로 합니다. MobileNet v2는 경량화된 depthwise 합성곱를 사용하여 중간 확장 계층의 특징 들을 필터링합니다. 또한, 표현력 유지를 위해 좁은 계층의 비선형성은 제거되었습니다.
```
Comment (spongebob03) [inline pytorch_vision_mobilenet_v2.md:85]: - 첫번째 문장의 원문이 길고 `반전된 잔차 구조`에 대한 수식어가 많아서 한번 읽고 이해하기 어려울거 같습니다. 주절과 잔차 구조에 대한 설명을 한번 끊어서 설명하는건 어떨지 의견 드립니다😀
- `특징 들을 필터링합니다` -> `특징들을 필터링합니다` 로 수정되면 될거같습니다:)

**Review by spongebob03 — COMMENTED** [review]
번역 잘해주셔서 특정 문단에 대한 수정 사항 코멘트 남겼습니다:D

File: pytorch_vision_mobilenet_v2.md (hunk 3 above)
Comment (mindongjun) [reply pytorch_vision_mobilenet_v2.md]: 복잡한 문장을 보기 쉽게 정리할 수 있도록 제안주셔서 감사합니다.
긴 수식어가 있는 문장을 핵심 구조에 대해 이야기하고 해당 구조에 대한 설명부분으로 분리했습니다.
또한 띄어쓰기도 수정했습니다. 리뷰 감사합니다.

File: pytorch_vision_mobilenet_v2.md [hunk 4]
```diff
@@ -23,23 +23,22 @@ model = torch.hub.load('pytorch/vision:v0.10.0', 'mobilenet_v2', pretrained=True
 model.eval()
 ```
 
-All pre-trained models expect input images normalized in the same way,
-i.e. mini-batches of 3-channel RGB images of shape `(3 x H x W)`, where `H` and `W` are expected to be at least `224`.
-The images have to be loaded in to a range of `[0, 1]` and then normalized using `mean = [0.485, 0.456, 0.406]`
-and `std = [0.229, 0.224, 0.225]`.
+모든 사전 훈련된 모델들은 입력 이미지가 동일한 방식으로 정규화되었다고 상정합니다.
```
Comment (Taeyoung96) [inline pytorch_vision_mobilenet_v2.md:26]: `상정합니다.`가 혹시 어떤 뜻인지 여쭤봐도 될까요?  
`가정합니다.`를 잘못 쓰신건지 질문드립니다.

File: pytorch_vision_mobilenet_v2.md [hunk 5]
```diff
@@ -50,46 +49,46 @@ preprocess = transforms.Compose([
     transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
 ])
 input_tensor = preprocess(input_image)
-input_batch = input_tensor.unsqueeze(0) # create a mini-batch as expected by the model
+input_batch = input_tensor.unsqueeze(0) # 모델에서 상정하는 미니배치 생성
```
Comment (Taeyoung96) [inline pytorch_vision_mobilenet_v2.md:52]: 여기도 `상정하다`는 이야기가 있네요..  
이쯤 되면 제 국어 실력에 문제가 있는 것 같습니다..ㅎ

**Review by Taeyoung96 — COMMENTED** [review]
`상정하다`라는 단어가 익숙치 않아서 comment 남겼습니다..ㅎ

File: pytorch_vision_mobilenet_v2.md (hunk 4 above)
Comment (mindongjun) [reply pytorch_vision_mobilenet_v2.md:26]: 안녕하세요 @Taeyoung96 님
해당 내용은 분명하지 않은 것을 임시로 인정하는 '가정'의 의미보다는 해당 모델은 정규화가 되었다는 사실을 딱 잘라서 판단하고 결정한 '상정'의 의미가 문맥 상 더 적합한 의미라고 판단하여 작성했습니다. 하단의 국립국어원의 관련 내용 링크로 첨부합니다.
그런데 둘다 한자어로 번역한 것이기 때문에 다른 순한글어 번역어에 대해서도 한번 찾아보겠습니다.

> ‘상정’은 ‘딱 잘라서 판단하고 결정함’이라는 ‘단정’의 뜻이, ‘가정’은 ‘(임시로) 그렇다고 여김’이라는 ‘인정’의 뜻이 들어 있으므로, 이러한 뜻 차이를 고려하셔서 문맥에 맞게 ‘상정’이나 ‘가정’을 쓰시면 되겠습니다.
>
> 상정(想定)
>
> 「명사」
>
> 어떤 정황을 가정적으로 생각하여 단정함. 또는 그런 단정.
>
> ¶ 이 우스꽝스러운 상정을 남들이 일소에 부칠지라도 그때의 나로서는 꽤나 심각한 상정이었다.≪윤후명, 별보다 멀리≫
>
> 가정(假定)
>
> 「명사」
>
> 「1」사실이 아니거나 또는 사실인지 아닌지 분명하지 않은 것을 임시로 인정함.
>
> ¶ 그의 의식 속에는 만약이라는 가정이 항상 존재하고 있다./6월 초 선거가 실시된다는 가정 아래 준비를 해 왔다.

관련 국립국어원 내용 링크로 남겨드립니다.

https://www.korean.go.kr/front/onlineQna/onlineQnaView.do?mn_id=216&qna_seq=96465

File: pytorch_vision_mobilenet_v2.md (hunk 5 above)
Comment (mindongjun) [reply pytorch_vision_mobilenet_v2.md:52]: 아닙니다. '가정하다','상정하다' 모두 한자어기 때문에 순한글어로 더 적합한 뜻이 있는지 찾아보겠습니다.

File: pytorch_vision_mobilenet_v2.md (hunk 4 above)
Comment (codingbowoo) [reply pytorch_vision_mobilenet_v2.md:26]: 여전히 한자어지만, `요구하다` 라는 말이 조금 더 익숙하지 않나 싶습니다.  
All pre-trained models expect input images normalized in the same way,

```suggestion
모든 사전 훈련된 모델들은 입력 이미지를 동일한 방식으로 정규화할 것을 요구합니다.
```

이런 의역도 가능하지 않을까요? 
```suggestion
모든 사전 훈련된 모델들은 동일한 방식으로 정규화된 이미지를 입력으로 사용합니다.
```

File: pytorch_vision_mobilenet_v2.md (hunk 5 above)
Comment (codingbowoo) [reply pytorch_vision_mobilenet_v2.md:52]: 여기도 마찬가지로 적당한 의역을 하면 어떨까요?
```suggestion
input_batch = input_tensor.unsqueeze(0) # 모델에서 요구하는 형태의 미니배치 생성
```

File: pytorch_vision_mobilenet_v2.md (hunk 4 above)
Comment (mindongjun) [reply pytorch_vision_mobilenet_v2.md]: 안녕하세요. 아 네넵 마지막 의역이 자연스러운 번역으로 읽히는 것 같습니다.
반영하도록 하겠습니다!

File: pytorch_vision_mobilenet_v2.md (hunk 5 above)
Comment (mindongjun) [reply pytorch_vision_mobilenet_v2.md]: 안녕하세요 해당 번역 제안사항으로 업데이트 했습니다.
제안주셔서 감사합니다 :)

**Review by Taeyoung96 — APPROVED** [review]
의견 반영해주셔서 감사드립니다. :smile:
